projects/netgrif-components-core/src/lib/search/models/category/case/case-title.spec.ts

```diff
@@ -64,4 +70,10 @@ describe('CaseTitle', () => {
             done();
         });
     });
+
+    it('should use keyword index', () => {
+        configureCategory(category, operatorService, Substring, ['foo']);
+        const query = resolver.getCoreIndex(CaseSearch.TITLE, category.selectedOperator === operatorService.getOperator(Substring));
```

Comment:
This does not check the implementation in the category class, but in the resolver. The `generateQuery()` method of the `Category` class should be used to generate the query

Reply:
`generateQuery()` is protected method and cannot be called from test. It calls the `elasticKeywords()` function which calls the same function with same attributes as above.